### ISSUE TITLE
Minting more than one x the token is not working

### ISSUE THREAD

**Issue by OurMAVEN** [tokenocean/maven#12]
We went to mint a multiple of 10 Giant Sea Turtle Tokens.

Minting seemed to work but when we go to list them, they are not able to list.
The token is not being found.

I had to delete 9 out of 10 and manually re mint.

I am not sure why the multiplier is not working.
- Image sizes are a lot smaller so this should not be an issue

For now I will continue to list one at a time, but not efficient.

**Comment by secondl1ght** [comment]
I tested this in prod and was not able to mint multiple tokens, the UI said it was successful but under the Creations tab on my profile I only see edition 1 of 5, same with the assets in my wallet. I tried listing this one and it again said it was successful but there was still a List button showing after, so I am not sure if it worked.

Going to assign his back over to Adam to fix these backend issues.
